### PR TITLE
[issueid # JAV-43] spring boot tranport demo code optimized and remov…

### PR DESCRIPTION
**Pull request by acsukesh**
…ed unused code like highway,tcc..

Signed-off-by: sukesh a c <sukeshac@huawei.com>

### CODE REVIEW

**Comment by seanyinx** [comment]
is this demo passing with the fix?

**Comment by acsukesh** [comment]
yes , its fixed

**Comment by seanyinx** [comment]
could you please add an automated integration test for it? you may refer to other demos, such as crossapp, jaxrs, or spring mvc.

**Comment by acsukesh** [comment]
verify and fix the build issues first before pull request

File: demo/demo-spring-boot-transport/demo-spring-boot-pojo-client/src/test/java/io/servicecomb/springboot/pojo/client/PojoClientIT.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+public class PojoClientIT {
+
+    @Before
+    public void setUp() throws Exception {
+        TestMgr.errors().clear();
+    }
+
+    @Test
+    public void clientGetsNoError() throws Exception {
+        PojoClient.main(new String[0]);
```
Comment (seanyinx) [inline demo/demo-spring-boot-transport/demo-spring-boot-pojo-client/src/test/java/io/servicecomb/springboot/pojo/client/PojoClientIT.java]: please start spring application with the following command to fix the test error
```
SpringApplication.run(PojoClient.class);
```

**Pull request closed without merging**